src/vendor/wordpress/_wordpress.scss

```diff
@@ -55,6 +55,57 @@ $wp-button-gap: sizes.$list-inline-gap;
   }
 }
 
+/**
+ * Gutenberg block: File download button
+ * Applies our pattern library styles to gutenberg file download blocks.
+ */
+
+.wp-block-file {
+  /**
+   * 1. Prevents stacked file buttons from collapsing on each other. 
+   */
+
+  + .wp-block-file {
+    margin-top: $wp-button-gap; /* 1 */
+  }
+
+  /**
+   * 1. Prevents default WP styling from modifying buttons opacity.  
+   */
+
+  .wp-block-file__button,
+  :not(.wp-block-file__button) {
```

Comment:
Could you change `:not(.wp-block-file__button)` to `*` and add the same comment explaining why here?